src/index.ts

```diff
@@ -136,9 +139,9 @@ export class BlockSuggestor {
     if (
       e.type === Blockly.Events.BLOCK_CREATE &&
       this.workspaceHasFinishedLoading &&
-      e.json?.type
+      (e as any).json?.type
```

Comment:
Although this suppresses the type error (`Property 'json' does not exist on type 'Abstract'`) here, notice there's a warning about using `any`. The issue is it's not necessarily a property on `Blockly.Events.Abstract`, but if it gets to this point, we know it's of type `Blockly.Events.BLOCK_CREATE` and `json` is an optional property on these types of objects (See: https://developers.google.com/blockly/reference/js/blockly.events_namespace.blockcreate_class.md). So, you'll want to update your casting, accordingly.